### PR TITLE
display page content with forward and backward button working

### PR DESCRIPTION
**Pull request by hammour**
- Added story content (page 'body') to page
- Added story image
- Click events for moving forwards and backwards
- Keep track of current page

### CODE REVIEW

**Comment by willcook4code** [comment]
👍

File: public/scripts/components/pages/Read.js [hunk 1]
```diff
@@ -35,19 +39,29 @@ export default React.createClass({
...
+					<img className="eleven columns" src={this.state.story.get('pages')[this.state.currentPage].image}/>
+					<p className="eleven columns">{this.state.story.get('pages')[this.state.currentPage].body}</p>
 					<div className="eight columns directionals-container">
-						<button className="directionals">Back</button>
-						<button className="directionals">Forward</button>
+						<button className="directionals" onClick={this.previousPage}>Back</button>
+						<button className="directionals" onClick={this.nextPage}>Forward</button>
 					</div>
-					<h1>Pg. 1</h1>
+					<h1>Pg. {this.state.currentPage}</h1>
```
Comment (alarner) [inline public/scripts/components/pages/Read.js]: Will this display page zero? @hammour

**Comment by alarner** [comment]
👍

**Comment by willcook4code** [comment]
👍

**Comment by judan** [comment]
💯

**Pull request merged**